dashboard/config/environment.rb

```diff
@@ -11,3 +11,5 @@
 
 # Initialize the Rails application.
 Dashboard::Application.initialize!
+
+ActiveRecord::SchemaDumper.ignore_tables = ['overflow_activities']
```

Comment:
Probably worth adding a comment why this needs to be ignored (in certain environments only), why it can't be committed to GitHub, etc.